### DISCUSSION TITLE
EFB charts

### DISCUSSION THREAD

**Discussion by rasse622** [magknight/787aviatorsTracker#522] [Ideas]
![0BC1A5A1-F6CB-402C-9799-661582C69478](https://user-images.githubusercontent.com/63057989/80623906-98e0a800-8a53-11ea-92b8-b0d644bbf134.jpeg)

**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by aeroniemi** [comment]
there is avitab, which does kinda do this
